Shopify/dawn
# [Cards and Buttons] "Vertical lift" effect on hover

### PR Summary: 

Adds a simple "raise" effect to buttons and cards on hover. 

### Why are these changes introduced?

Closes https://github.com/Shopify/dawn/issues/2574. 

### What approach did you take?

Piggybacks on the work done in https://github.com/Shopify/dawn/pull/2452. Uses the same control in Theme Settings, and applies a different set of rules to just simply raise the elements on hover. 

### Visual impact on existing themes

When merchants visit Theme Settings > Animation, and choose the "Vertical lift" hover effect, cards and buttons will each. get a little "bump" effect on hover: 

https://user-images.githubusercontent.com/1202812/234607022-9aacddad-b9bd-4355-94db-4d3dc7d8d8ec.mp4

### Testing steps/scenarios

- Visit the Theme Settings > Animations
- Select "Vertical lift" from the "Hover Effect" dropdown. 
- Note that the raise effect is applied to buttons and cards throughout the storefront. 

### Demo links

- [Store](https://os2-demo.myshopify.com/?preview_theme_id=139846287382)
- [Editor](https://os2-demo.myshopify.com/admin/themes/139846287382/editor)

### Checklist
